pkg/apis/imageregistry/v1alpha1/types.go

```diff
@@ -59,6 +59,8 @@ type ImageRegistryConfigStorageFilesystem struct {
 	VolumeSource corev1.VolumeSource `json:"volumeSource,omitempty"`
 }
 
+type ImageRegistryStorageType string
```

Comment:
Why is it a part of our public API?

Reply:
+1, @coreydaley please move this out of the api

Reply:
updated in current push